config/smartcosmos-ext-interactions.yml

```diff
@@ -0,0 +1,16 @@
+server:
+  port: ${smartcosmos.service.ext-interactions.port}
+
+smartcosmos:
+  security:
+    enabled: true
+  endpoints:
+    interactions:
+      enabled: true
+      create.enabled: true
+  interactions:
+    kafka:
+      host: ${smartcosmos.service.kafka.address}
+      port: 9092
+      address: ${smartcosmos.service.kafka.host}:${smartcosmos.service.kafka.port}
+
```

Comment:
updated this also in https://github.com/SMARTRACTECHNOLOGY/smartcosmos-ext-interactions/pull/8